src/hooks/useEmails.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  useQuery,
+  type QueryFunction,
+  type QueryKey,
+  useMutation,
+} from '@tanstack/react-query';
+import axios, { type AxiosError } from 'axios';
+import { EMAILAPI_URL } from '../constants';
+import { type Email, type EmailData, type SendEmailsResponse } from '../types';
+
+const fetchEmails: QueryFunction<Email[], QueryKey> = async () => {
+  const response = await axios.get(`${EMAILAPI_URL}/api/v1/sent`);
+  return response.data.emails;
+};
+
+const sendEmail = async (emailData: EmailData): Promise<SendEmailsResponse> => {
+  const response = await axios.post(`${EMAILAPI_URL}/api/v1/send`, emailData);
+  return response.data;
+};
+
+const useSendEmail = () => {
```

Comment:
Add this inside the useEmails hook itself, go through this https://github.com/sef-global/scholarx-frontend/blob/main/src/hooks/admin/useMentees.ts